chore: add warning to local monitoring page (#410)
Substra/substra-documentation: docs/source/how-to/using-substra/get_performances.rst

How to monitor performance in local mode
========================================

> .. warning::
>     This page has been tested with Substra 0.36.0. Some changes might be needed is your using a more recent version.
> 

Reviewer: ```suggestion
.. warning::
    The last time this page was tested was with Substra 0.36.0. Some changes might be needed if you are using a more recent Substra version.

```
